server/package.json

```diff
@@ -15,7 +15,7 @@
   "scripts": {
     "build": "cd .. && cd client && npm run build && cd .. && cd server",
     "start": "ASSET_COMPUTE_DEV_TOOL_ENV=development concurrently \"nodemon bin/www\" \"npm run client\"",
-    "server": "ASSET_COMPUTE_DEV_TOOL_ENV=production node bin/www",
+    "server": "set ASSET_COMPUTE_DEV_TOOL_ENV=production && node bin/www",
```

Comment:
we need this to work in all OS. Just like we have a `cross-os-build`, we need a `cross-os-run-server`